main_test.go

```diff
@@ -19,15 +34,14 @@ func TestMain(m *testing.M) {
 
 func setupIntegrationTests() {
 	flag.Parse()
-	*tests.RtUrl = utils.AddTrailingSlashIfNeeded(*tests.RtUrl)
 
 	if *tests.TestBintray {
 		InitBintrayTests()
 	}
 	if *tests.TestArtifactory && !*tests.TestArtifactoryProxy {
 		InitArtifactoryTests()
```

Comment:
Inside the *InitArtifactoryTests* function, the first if statement has now become redundant.